website/content/en/status/report-2024-01-2024-03/cloud-init.adoc

```diff
@@ -0,0 +1,34 @@
+=== FreeBSD as a Tier 1 cloud-init Platform
+
+Links: +
+link:https://cloud-init.io/[cloud-init Website] URL: link:https://cloud-init.io/[] +
+link:https://cloudinit.readthedocs.io/en/latest/[cloud-init Documentation] URL: link:https://cloudinit.readthedocs.io/en/latest/[] +
+
+Contact: Mina Galić <freebsd@igalic.co>
+
+cloud-init is the standard way of provisioning servers in the cloud.
+Over the past year and a half, thanks to this FreeBSD support has steadily improved.
+This year, we took a pivot and decided to explicitly focus on making improvements in FreeBSD itself, that will aid the cloud-init team to test future changes to FreeBSD code-paths themselves.
+To achieve this goal, we need to make FreeBSD run in LXD (and Incus), which it already can, but not under control of ``lxd-agent`` (or ``incus-agent``).
+
+Here are some improvements from the recent weeks:
+
+- I have written a small https://codeberg.org/meena/test-cloud-init[testing-framework] (in sh, and I'm slowly porting it to OpenTofu/Terraform), which installs the latest version of package:net/cloud-init-devel[] or package:net/cloud-init[] and runs a couple of standard cloud-init tests.
+- To do this, I have created a https://pkg.igalic.co/[dedicated public repository] which contains the latest versions of package:net/cloud-init-devel[] and package:net/cloud-init[] for FreeBSD 13 and 14 on amd64 and aarch64
+- I have ported Linux' vsock testing framework to https://codeberg.org/meena/vsock-tests[FreeBSD]
```

Comment:
I'd move some of the surrounding text into the link text, for better readability when links are shown separate. Something like:

> - I have https://codeberg.org/meena/vsock-tests[ported Linux' vsock testing framework to FreeBSD]

Reply:
I am not a native English speaker so I might be wrong, but I am not used to see apostrophes at the end of words ending in `x`. Is `Linux' vsock testing framework` correct? Shouldn't it be `Linux vsock testing framework` or `Linux's vsock testing framework`? Is there some rule saying that words ending in `x` act as words finishing in `s`?

Reply:
fixed, honestly don't know where I learned that.